tuxemon/menu/quantity.py

```diff
@@ -4,6 +4,11 @@
 from typing import Any, Callable, Generator, Optional
 
 from tuxemon.locale import T
+#----------Showing money when buying/selling---------------
+from tuxemon.locale import gettingMoney
```

Comment:
The two locale imports should be on the same line, the session import should be later (alphabetical order(